discord/flags.py

```diff
@@ -1198,3 +1198,39 @@ def gateway_message_content_limited(self):
         and has hit the guild limit.
         """
         return 1 << 19
+
+
+@fill_with_flags()
+class ChannelFlags(BaseFlags):
+    r"""Wraps up the Discord Channel flags.
```

Comment:
Why the r"""

Reply:
I just copy/pasted from another flag class, they're all like that (except for MemberCacheFlags)

Reply:
It treats the docstring as a raw string (i.e. escape characters are ignored and treated as text). It's fine to leave as-is.